src/test/run-make/wasm-stringify-ints-small/foo.rs

```diff
@@ -0,0 +1,39 @@
+#![crate_type = "cdylib"]
+
+extern "C" {
+    fn observe(ptr: *const u8, len: usize);
+
+    fn get_u8() -> u8;
+    fn get_i8() -> i8;
+    fn get_u16() -> u16;
+    fn get_i16() -> i16;
+    fn get_u32() -> u32;
+    fn get_i32() -> i32;
+    fn get_u64() -> u64;
+    fn get_i64() -> i64;
+    fn get_usize() -> usize;
+    fn get_isize() -> isize;
+}
+
+macro_rules! stringify {
```

Comment:
I would *personally* rename this to avoid potential (accidental) conflicts with the language-provided stringify.

Reply:
Agreed -- I forgot about that provided macro.